runtime/gc_modron_startup/mmhelpers.cpp

```diff
@@ -391,7 +391,15 @@ j9gc_modron_getConfigurationValueForKey(J9JavaVM *javaVM, UDATA key, void *value
 		*((UDATA *)value) = 0;
 		keyFound = FALSE;
 #endif /* defined(OMR_GC_VLHGC_CONCURRENT_COPY_FORWARD) */
-
+		break;
+	case j9gc_modron_configuration_gcConcurrentMarkThreadCount:
+#if defined(OMR_GC_MODRON_CONCURRENT_MARK)
```

Comment:
should we have a check for GC policy too?
this is only used for gencon and optavgpause
it's set to 1 regardless of the policy, so it would be incorrect to return 1 for optthruput or balanced.

for balanced we should probably return the number of total threads (mark increments are scheduled as parallel tasks with all available threads (questionable if it's optimal, but that's the state right now))